data_dir/c-s_lwt_basic.yaml

```diff
@@ -6,14 +6,14 @@ keyspace: cqlstress_lwt_example
 
 # The CQL for creating a keyspace (optional if it already exists)
 keyspace_definition: |
-  CREATE KEYSPACE cqlstress_lwt_example WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+  CREATE KEYSPACE IF NOT EXISTS cqlstress_lwt_example WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
```

Comment:
Change all irrelevant comments in this file, they are misleading.

Reply:
Changed